colandr/extensions.py

```diff
@@ -21,6 +21,12 @@ class _BaseModel(sqlalchemy.orm.DeclarativeBase):
 mail = flask_mail.Mail()
 migrate = flask_migrate.Migrate()
 
+review_model_cache = flask_caching.Cache(config={
+    'CACHE_TYPE': 'SimpleCache',
+    'CACHE_DEFAULT_TIMEOUT': 3600 * 24,  # 24 hours
+    'CACHE_THRESHOLD': 25  # Maximum number of items
+})
```

Comment:
nit: auto-format for style consistency

```suggestion
review_model_cache = flask_caching.Cache(
    config={
        "CACHE_TYPE": "SimpleCache",
        "CACHE_DEFAULT_TIMEOUT": 3600 * 24,  # 24 hours
        "CACHE_THRESHOLD": 25  # Maximum number of items
    }
)
```

Reply:
done